core/src/main/java/kafka/server/share/SharePartitionManager.java

```diff
@@ -250,13 +250,39 @@ public CompletableFuture<Map<TopicIdPartition, ShareAcknowledgeResponseData.Part
         String groupId,
         Map<TopicIdPartition, List<ShareAcknowledgementBatch>> acknowledgeTopics
     ) {
-        log.trace("Acknowledge request for topicIdPartitions: {} with groupId: {}",
+        log.debug("Acknowledge request for topicIdPartitions: {} with groupId: {}",
```

Comment:
Any reason to change this?

Reply:
Nope, it was a mistake, thanks for pointing it out